docs/community/governance.md

```diff
@@ -95,8 +100,8 @@ then admission to the SC by consensus. During that time deadlocked votes of the
 be postponed until the new member has joined and another vote can be held. The IFP seat
 is elected by the IFP Advisory Council.
 
-The SC may be contacted at `napari-steering-council@googlegroups.com`.
-Or via the [@napari/steering-council](https://github.com/orgs/napari/teams/steering-council) GitHub team. For a list of the current SC see our [About Us](team) page.
+The SC may be contacted at `napari-steering-council@googlegroups.com`. For a list of the current 
```

Comment:
Were we switching to a (at)napari.org adress for this?

Reply:
@psobolewskiPhD I think we decided not to (at least for forseeable future) because of the issues with administering private groups or something?